pkgs/servers/monitoring/prometheus/nginx_exporter/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, goPackages, fetchFromGitHub }:
+
+goPackages.buildGoPackage rec {
+  name = "prometheus-nginx-exporter-${builtins.substring 0 7 rev}";
```

Comment:
Are you aware of [this PR](https://github.com/NixOS/nixpkgs/pull/6539)?

(If so, fine. I'm not the version police.)

(Actually that sounds cool and I am now the version police. :cop: )